package.json

```diff
@@ -47,7 +47,7 @@
     "webpack-merge": "^0.14.1"
   },
   "peerDependencies": {
-    "chartist": "^0.10.1",
+    "chartist": "^0.11.0",
```

Comment:
Alternatively, this could be `"chartist": ">=0.10.1 <1.0.0"` if we we don't expect chartist to break their API until v1.